src/audio/tdfb/tdfb_direction.c

```diff
@@ -0,0 +1,497 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
+
+//#define DEBUG
+
+#include <ipc/topology.h>
+#include <sof/audio/eq_iir/iir.h>
+#include <sof/audio/tdfb/tdfb_comp.h>
+#include <sof/lib/alloc.h>
+#include <sof/math/iir_df2t.h>
+#include <sof/math/trig.h>
+#include <user/eq.h>
+#include <stdint.h>
+#include "tdfb_math.h"
+
+#ifdef DEBUG
+#include <stdio.h>
+FILE *fh;
```

Comment:
What's this needed for ?

Reply:
Removed and fixed. All internals testing is based on verbose mode traces now.